index.bs

```diff
@@ -1457,6 +1459,79 @@ Resignation from a Group</h3>
 	See “Exclusion and Resignation from the Working Group” in the W3C Patent Policy [[PATENT-POLICY]]
 	for information about obligations remaining after resignation from certain groups.
 
+<h3 id="tooling">
+Tooling</h3>
+
+	For [=W3C Groups=] operating under this Process,
+	a core operating principle is to allow access across disabilities,
+	across country borders,
+	and across time.
+	Thus in order to allow all would-be participants to effectively participate,
+	to allow future participants and observers to understand the rationale and origins of current decisions,
+	and to guarantee long-lived access to its publications,
+	W3C requires that:
+
+	<ul>
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			(i.e. intended for use or reference outside its own membership)
+			<em class=rfc2119>must</em> be published and promoted at a W3C-controlled URL,
+			and backed up by W3C systems
+			such that if the underlying service is discontinued,
+			W3C can continue to serve such content without breaking incoming links
+			or other key functionality.
+
+		<li>
+			All reports, publications, or other deliverables
+			produced by the group for public consumption
+			<em class=rfc2119>must</em> follow best practices for accessibility worldwide
+			and for persons with disabilities.
+			Network access to w3.org itself may be assumed.
+
+		<li>
+			Persistent text-based discussions sponsored by the group
+			pertaining to their work
+			and intended to be referenceable by all group members,
+			including official meeting minutes and especially records of decisions made,
+			<em class=rfc2119>must</em> be archived by W3C for future reference.
+			This includes discussions conducted over email lists
+			or in issue-tracking services
+			or any equivalent fora.
+			Any discussion system whose archive is separate from its active interactive operation
+			<em class=rfc2119>must</em> make that archive available
+			at the same level of confidentiality.
+
+		<li>
+			Any tooling used by the group
+			for producing its documentation and deliverables
+			or for official group discussions
+			<em class=rfc2119>must</em> be usable
+			without cost (direct or indirect)
+			by all active or would-be active group participants
```

Comment:
is it clear what "would-be active" means? maybe a way to think about the phrasing is to figure out who needs to be explicitly consulted before a tool can be adopted; in the same line as above, another question is what recourse might such a would-be active participant have if the tool that is adopted doesn't follow that rule

Reply:
The recourse in general is to complain, cite this rule, and escalate as needed. I agree, would-be active needs work.

Reply:
Would-be active is some actual person who would be active if the tooling barrier to participation were removed. It's intended not apply to a theoretical person or theoretical participation. If you've got a better way to phrase that...

Reply:
"all active participants, or those whose activity is curtailed because of a tooling choice."